### PR TITLE
Add links page

### PR DESCRIPTION
**Pull request by gavinmak**
Add a simple page for students to access their links (homework, jamboard, etc.)

### CODE REVIEW

**Comment by gavinmak** [comment]
Dropdown after "Rework shortening" commit:

Screenshot 2023-09-22 at 11 40 19 AM

File: client/links.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+      }
+    }
+
+    truncLen++;
+  }
+
+  return ret;
+}
+
+function titleCase(s) {
```
Comment (GregoryCannon) [inline client/links.js]: toTitleCase

File: client/links.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  const words = s.split(' ');
+  return words.map(
+    w => w[0].toUpperCase() + w.substring(1).toLowerCase()
+  ).join(' ');
+}
+
+// Returns a clickable DOM element that navigates to the link.
+function linkElement(link) {
```
Comment (GregoryCannon) [inline client/links.js]: getLinkElement

**Review by GregoryCannon — APPROVED** [review]

File: client/links.js (hunk 1 above)
Comment (gavinmak) [reply client/links.js]: renamed

File: client/links.js (hunk 2 above)
Comment (gavinmak) [reply client/links.js]: renamed

**Comment by gavinmak** [comment]
I don't have write access, can you help merge this?

**Pull request merged**